third-culture-software/bhima
# Display asset info in lot registry

Allow the display of asset info in the lot registry.  Also support editing the asset serial number in the lot editor modal.

Finishes items in Issue https://github.com/IMA-WorldHealth/bhima/issues/6349

**TESTING**
- Use any dataset
- Use the Stock > articles in Stock to find an inventory item with articles in stock.
- Edit the inventory item for the lot from the last step and change it to an asset.  Fill out the other asset related fields (manufacturer brand, model, etc).  Then save
- Go back to the lots in stock for the inventory item modified above
- Use [Menu] > Change Columns to display the asset related info (manufacturers info, serial number, etc).
- Verify that the asset-related fields are shown in the lot registry grid display
- Use the action menu to edit and save the asset and set its serial number
- Verify that the serial number just added is displayed.   
- Edit the asset again and change the serial number.  Verify it is changed in the grid.
- Revert the inventory item to it prior 'asset' status (or rebuild the database)
